documentation/sphinx/source/release-notes/release-notes-630.rst

```diff
@@ -6,6 +6,7 @@ Release Notes
 ======
 * Several minor problems with the versioned packages have been fixed. `(PR 5607) <https://github.com/apple/foundationdb/pull/5607>`_
 * A client might not honor transaction timeouts when using the multi-version client if it cannot connect to the cluster. `(Issue #5595) <https://github.com/apple/foundationdb/issues/5595>`_
+* Fixed a very rare bug where recovery could potentially roll back a committed transaction `(PR 5461) <https://github.com/apple/foundationdb/pull/5461>`_
```

Comment:
I just noticed a minor nit, which is that the release notes generally include periods at the end of the sentence:

```suggestion
* Fixed a very rare bug where recovery could potentially roll back a committed transaction. `(PR 5461) <https://github.com/apple/foundationdb/pull/5461>`_
```